README.md

```diff
@@ -157,8 +157,8 @@ Join our weekly Zoom meetings:
 
 https://zoom.us/j/3595189922
 
-- Office hours Mondays at 5:00 UTC
-- Maintainer hours Thursdays at 5:00 UTC
+- Office hours Mondays at 17:00 UTC
+- Maintainer hours Thursdays at 17:00 UTC
```

Comment:
I think it'd also be helpful to include a link to that everytimezone.com bit.

Reply:
Agree.